Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,134 @@
+// swiftlint:disable all
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+struct CreditCardPOSTBody: Encodable {
+    var authenticationInsight: Bool?
+    var merchantAccountID: String?
+    var meta: Meta?
+    let creditCard: CreditCard?
+    
+    enum CodingKeys: String, CodingKey {
+        case authenticationInsight
+        case meta = "_meta"
+        case merchantAccountID = "merchantAccountId"
+        case creditCard = "credit_card"
+    }
+
+    init(
+        card: BTCard,
+        metaData: BTClientMetadata
```

Comment:
nit since this is the casing we use elsewhere:
```suggestion
        metadata: BTClientMetadata
```